zerver/tests/test_markdown.py

```diff
@@ -1156,6 +1158,58 @@ def test_fetch_tweet_data_settings_validation(self) -> None:
+        expected = f"""<div class="kroki-block">{html}</div>"""
+        assert_kroki_diagrams(msg, expected, html)
+
+        # Kroki will not be able to generate html string unless diagram type is not provided.
```

Comment:
I don't understand this comment.

Reply:
Kroki cannot automatically guess the diagram type (when we don't provide it while making the POST request) -- hence it'll not generate an HTML string. (The expected output clarifies the same)